app/experimenter/static/css/experimenter.css

```diff
@@ -182,3 +182,11 @@ pre.json {
   line-height: 2em;
   border: none !important;
 }
+
+.range-field {
```

Comment:
You don't need to add custom CSS rules, bootstrap has these handy margin classes so you can just say

    class="mb-1"

We use them all over the place, docs here:

https://getbootstrap.com/docs/4.0/utilities/spacing/